lib/simp/metadata/commands/component.rb

```diff
@@ -8,18 +8,18 @@ def run(argv, engine = nil)
 
           case subcommand
           when '--help', '-h'
-            options = defaults(argv) do |opts,options|
+            options = defaults(argv) do |opts, options|
               opts.banner = 'Usage: simp-metadata component [ view | diff | download | build | update | create ]'
             end
 
           when 'create'
-            options = defaults(argv) do |opts,options|
+            options = defaults(argv) do |opts, options|
               opts.banner = 'Usage: simp-metadata component create <component_name> name=<value>'
             end
             engine, root = get_engine(engine, options)
             component = argv[1]
             argv.shift
-            data = {'locations' => [{'primary' => true}]}
+            data = {:locations => [{:primary => true}]}
```

Comment:
See first comment above. This is probably fine, as long as it's the same as whatever you choose above.